_layouts/default.html

```diff
@@ -9,6 +9,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta name="theme-color" content="#157878">
     <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
+    <link href="https://unpkg.com/prismjs@1/themes/prism.css" rel="stylesheet">
```

Comment:
Be consistent with rel first?
```suggestion
    <link rel="stylesheet" href="https://unpkg.com/prismjs@1/themes/prism.css">
```